improver_tests/acceptance/SHA256SUMS

```diff
@@ -633,73 +633,73 @@ e653ae271dd789dde8b03e6237cc7b48552dd9d5206d3430ee68d283683247a0  ./wind_downsca
+ce6a854694aecd515063c91774eeb6160f39c9dbf5e970f72f960a3527e157e9  ./wxcode-modal/spot_input/20201209T1000Z-weather_symbols-PT01H.nc
+300c38e7f522ff1e9257708b183148f3441bb1a861c15eaff2a5f1713bbbc872  ./wxcode-modal/spot_input/20201209T1100Z-weather_symbols-PT01H.nc
+48ef16ad1fd0b499f1d38f585ddb0a4299c39e7d7df625ad0fa6c3dc24e0f094  ./wxcode-modal/spot_input/20201209T1200Z-weather_symbols-PT01H.nc
+5ab6b249ec615ddab9995629aaddbd85bba352ad52b7bcb0be5b3037d75c22ca  ./wxcode-modal/spot_input/20201209T1300Z-weather_symbols-PT01H.nc
+340205e12625ff9466c1afba8c007343aa40e21b837ab2f2143fa032e6c47bd3  ./wxcode-modal/spot_input/20201209T1400Z-weather_symbols-PT01H.nc
+776739d288b6e43b3d9341ba301feead160986d2540514f0663f243ca003e55a  ./wxcode-modal/spot_input/20201209T1500Z-weather_symbols-PT01H.nc
+4655358c22abdd0acfd5f4e08f156a7f4f629a1764dd8314f28f121964725078  ./wxcode-modal/spot_input/20201209T1600Z-weather_symbols-PT01H.nc
+28ee3f9d419bb9766cc68e1c74fa9e8e5010dc63a600635a54356a33795f6644  ./wxcode-modal/spot_input/20201209T1700Z-weather_symbols-PT01H.nc
+073fd0175e35afcf7f9fe61fedfa63bbd9ea5299f68d8b0287bc30085339b942  ./wxcode-modal/spot_input/20201209T1800Z-weather_symbols-PT01H.nc
+2041f9ed3d1c350518dab5d537652165e58bbe92a003f5c2dfaa0ff94ccce1e1  ./wxcode-modal/spot_input/kgo.nc
+db1612786b08a929ad096f13ca603f37cfd78de681e1306d909654f345f9f409  ./wxcode-modal/spot_ties/20201209T0700Z-weather_symbols-PT01H.nc
+b5beedd2fa6c58ecaaa51358b8b7abadddc1ad072af2d5141bbf0019b458e84e  ./wxcode-modal/spot_ties/20201209T0800Z-weather_symbols-PT01H.nc
+c92e6cb002b8ff8f185c097a55ff214793c7595667c8a7ba4afc1ae0f456c13e  ./wxcode-modal/spot_ties/20201209T0900Z-weather_symbols-PT01H.nc
+fd216b5dc3837202f3293f22e664d4fa623f7a349abda9ee8aef3fb19df3f660  ./wxcode-modal/spot_ties/20201209T1000Z-weather_symbols-PT01H.nc
+fbff18f7ef5c2e3d462cbc0d78de30aca7534a37f427472011f6fa1835ec8020  ./wxcode-modal/spot_ties/20201209T1100Z-weather_symbols-PT01H.nc
+129beabb8e059e46d3f00be1c5a5b8c1cdd44c4f1f08fc86a4249d4a0f854272  ./wxcode-modal/spot_ties/20201209T1200Z-weather_symbols-PT01H.nc
+063e6f151af8a05f4c2276995384fc4058571144a851953752653eff7886130c  ./wxcode-modal/spot_ties/20201209T1300Z-weather_symbols-PT01H.nc
+4d23d6d1f8861feeb288ec7b68e57e893fa2ffd633de1b25ab4d4bc4aa3316cc  ./wxcode-modal/spot_ties/20201209T1400Z-weather_symbols-PT01H.nc
+71aac305fce32fb389ea63822cd88511137cf1260c1c0a55e28c1a5fcbc91cab  ./wxcode-modal/spot_ties/20201209T1500Z-weather_symbols-PT01H.nc
+6fe4b1563f0fbb4579dba46b12654f983cd6486f39986d52231aad5d18acf508  ./wxcode-modal/spot_ties/20201209T1600Z-weather_symbols-PT01H.nc
+d506afe1572ded010805662f057926ba19367a8bf0c9f700380b6b63ab440399  ./wxcode-modal/spot_ties/20201209T1700Z-weather_symbols-PT01H.nc
+1c0c3b4fcc95ea2d5c9658ebd42cbee8a2043afa607843e0e9013d273f7fc460  ./wxcode-modal/spot_ties/20201209T1800Z-weather_symbols-PT01H.nc
+8e8b8daeff5cb0f77f8bd36d1e6eee24534f7e1af42cbb8e119a6a3df175e4fc  ./wxcode-modal/spot_ties/kgo.nc
```

Comment:
In this file, it seems like the units for the `wmo_id` and `met_office_site_id` has changed from `unknown` to `1`. I think that the units should actually be `no_unit`.